src/components/Card/Card.tsx

```diff
@@ -30,9 +30,12 @@ const Card = ({ title, image, id, shortUrl, sentiment, topic, updateUserSentimen
       >
         <p className="topic-text">{topic.toUpperCase()}</p>
         <h2 className="article-title cy-article-title">{title}</h2>
-        <img className="article-image cy-article-image" src={image} alt={title} />        
+        <img className="article-image cy-article-image" src={image} alt={title} />
       </a>
       <p className="abstract-text"> {abstract} </p>
+      <button onClick={ () => toggleSaved(id) } type="button">
```

Comment:
This button should change its content when the user saves the article. Maybe we can have a method that checks whether the the article is in the savedArticles array and conditionally renders the button depending on whether it is already saved or not.

Reply:
Also, could we add styling to this button. You can use [this icon](https://fonts.google.com/icons?selected=Material+Icons+Outlined:favorite_border:) to save and [this one](https://fonts.google.com/icons?selected=Material+Icons+Outlined:favorite:) to remove from saved.

Reply:
Fixed!